src/providers/ipa/ipa_deskprofile_rules_util.c

```diff
@@ -261,7 +264,11 @@ ipa_deskprofile_rules_create_user_dir(
         goto done;
     }
 
-    ret = sss_create_dir(domain_dir, shortname, 0600, uid, gid);
+    /* In order to read, create (and thus traverse) the directory, we need
```

Comment:
remove 'thus' and parens
Tracerse and create are unrelated, create requires only 'w', traverse requires only 'x'. You may need to do both, but there is no causal dependency.